ion/tokenizer.go

```diff
@@ -668,6 +678,27 @@ func (t *tokenizer) readEscapedChar(clob bool) (rune, error) {
 	return 0, &SyntaxError{fmt.Sprintf("bad escape sequence '\\%c'", c), t.pos - 2}
 }
 
+func (t *tokenizer) isProhibitedControlChar(c int) bool {
```

Comment:
t is unused here, so maybe worth dropping it and adding these functions to textutils.go?